snc-library.sh

```diff
@@ -170,19 +170,20 @@ function create_pvs() {
 
     for pvname in $(seq -f "pv%04g" 1 ${count}); do
         if ! ${OC} get pv "${pvname}" &> /dev/null; then
-            generate_pv "${pvdir}/${pvname}" "${pvname}" | ${OC} create -f -
+            generate_pv "${pvdir}/${pvname}" "${pvname}" > tmp_pv.yaml 
+            retry ${OC} create -f tmp_pv.yaml
         else
             echo "persistentvolume ${pvname} already exists"
         fi
     done
 
     # Apply registry pvc to bound with pv0001
-    ${OC} apply -f registry_pvc.yaml
+    retry ${OC} apply -f registry_pvc.yaml
```

Comment:
Wondering if it's possible to do something like
```
OC="retry ${OC}"
```
so that all calls to `oc` go through a retry. Alternatively, this could be a `run_oc` function which calls `retry ${OC} ...`. Might be cleaner.